docs/pywbemclicmdlineinterface.rst

```diff
@@ -269,3 +269,60 @@ following examples, an underscore ``_`` is shown as the cursor:
 The pywbemcli shell supports history across multiple invocations of the shell
 using <UP-ARROW>, <DOWN-ARROW>.
 The pywbemcli history is stored in ``~/.pywbemcli_history``.
+
+
+.. _`Error handling`:
+
+Error handling
+--------------
+
+Pywbemcli terminates with one of the following program exit codes:
+
+* **0 - Success**: The pywbemcli command has succeeded.
+
+* **1 - Error**: In such cases, pywbemcli aborts the requested operation and
+  displays one or more human readable error messages on standard error.
+
+<  If this happens for a command entered in interactive mode, the pywbemcli shell
+  is not terminated; only the command that failed is terminated.
+
+  Examples for errors reported that way:
+
+  * Local system issues, e.g. pywbemcli history file or connections file cannot
+    be written to.
+
+  * WBEM server access issues, e.g. pywbemcli cannot connect to or authenticate
+    with the WBEM server. This includes CIM errors about failed authentication
+    returned by the server.
+
+  * WBEM server operation issues, e.g. pywbemcli attempts to retrieve an
+    instance that does not exist, or the WBEM server encountered an internal
+    error. This will mostly be caused by CIM errors returned by the server,
+    but can also be caused by the pywbemcli code itself.
+
+* **1 - Python traceback**: In such cases, pywbemcli terminates during its
+  processing, and displays the Python stack traceback on standard error.
+
+  If this happens for a command entered in interactive mode, the pywbemcli shell
+  also terminates with a program exit code of 1.
+
+  These Python tracebacks should never happen and are always considered a
```

Comment:
Not completely true.  For an error in a mock-server python script we output the traceback to help the user find their problem.

Reply:
I agree. I looked at how it does that and it is an error message with multiple lines that include the traceback. So I'll clarify both the item about errors and the traceback statement you commented on.

BTW, I have opened issue #373 on the way this error is displayed, but I think we can defer that to 0.6.0.

Reply:
DONE